vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1157,14 +1145,7 @@ class RoomDetailFragment @Inject constructor(
             }
 
             override fun onRichContentSelected(contentUri: Uri): Boolean {
-                // We need WRITE_EXTERNAL permission
-                return if (checkPermissions(PERMISSIONS_FOR_WRITING_FILES, requireActivity(), writingFileActivityResultLauncher)) {
-                    sendUri(contentUri)
-                } else {
-                    roomDetailViewModel.pendingUri = contentUri
-                    // Always intercept when we request some permission
-                    true
-                }
+                return sendUri(contentUri)
```

Comment:
Did you test this on Sticker too? I am suspicious, why we were requesting permission in the first place?

Reply:
Yes, everything is fine